src/migrations/20190509005606-create-token.js

```diff
@@ -0,0 +1,34 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Tokens', {
+    id: {
```

Comment:
do we really need to create a table for a token, if yes why

Reply:
I think so. Because we need to blacklist invalid token